Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -463,32 +489,34 @@ - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
-    
-    SPTableViewCell* cell = [tableView dequeueReusableCellWithIdentifier:cellIdentifier];
 
-    if (!cell) {
-        // this shouldn't be needed, but offscreen collection view cells are sometimes called on
-        // and this can lead to an occasional crash
-        cell = [[SPTableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:cellIdentifier];
-    }
+    SPNoteTableViewCell* cell = [tableView dequeueReusableCellWithIdentifier:SPNoteTableViewCell.reuseIdentifier forIndexPath:indexPath];
```

Comment:
Another asterisk to tweak.

Reply:
It's been SO LONG!! updating, thank you!